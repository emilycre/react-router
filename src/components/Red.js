import React from 'react';

export default function Red() {
  const style = {
    backgroundColor: 'red',
    width: '200px',
    height: '200px',
    borderRadius: '50%',
    margin: '1em'
  };

  return <div style={style}></div>;
}
