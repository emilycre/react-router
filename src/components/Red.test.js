import React from 'react';
import { shallow } from 'enzyme';
import Red from './Red';

describe('Red Component', () => {
  it('Renders Red', () => {
    const wrapper = shallow(<Red />);
    expect(wrapper).toMatchSnapshot();
  });
});
