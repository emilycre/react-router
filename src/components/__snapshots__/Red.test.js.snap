// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Red Component Renders Red 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "borderRadius": "50%",
      "height": "200px",
      "margin": "1em",
      "width": "200px",
    }
  }
/>
`;
